src/containers/Dashboard/Tutorials/Tutorial.js

```diff
@@ -18,27 +58,14 @@ class Card extends Component {
         </div>
         <div className="row">
           <div className="tutorial-container">
-            <div className="interview-prep-tutorial">
-              <div className="tutorial-card shadow-lg ml-5 mr-4 mb-5">
-                <TutorialCard href={INTERVIEW_PREP_KIT} />
-              </div>
-            </div>
-            <div className="learn-code">
-              <div className="tutorial-card shadow-lg ml-5 mr-2 mb-5">
-                <TutorialCard href={STATISTICS} />
-              </div>
-            </div>
-            <div className="statistics">
-              <div className="tutorial-card shadow-lg ml-5 mr-4 mb-5">
-                <TutorialCard href={JAVASCRIPT} />
-              </div>
-            </div>
-            <div className="javascript">
-              <div className="tutorial-card shadow-lg ml-5 mb-5">
-                <TutorialCard href={CODE} />
-              </div>
+          {this.state.tutorial.map((tutoria, index) => (
+            <div className="interview-prep-tutorial" key={index}>
+            <div className="tutorial-card shadow-lg ml-5 mr-4 mb-5">
+              <TutorialCard heading={tutoria}/>
```

Comment:
same here why just only name to tutorial

Reply:
Done.